ocs_ci/utility/utils.py

```diff
@@ -366,7 +366,9 @@ def run_cmd(cmd, **kwargs):
         (str) Decoded stdout of command
 
     """
-    log.info(f"Executing command: {cmd}")
+    silence_logs = kwargs.pop('silent', False)
+    if not silence_logs:
```

Comment:
Could you please create a function to silent the logs?
This approach is very hard to maintain with if/esle all over:

ipdb> import sys 
ipdb> logging.info("Before disable")                                                                                                                                                    
08:33:00 - MainThread - root - INFO - Before disable                                                                                                                                                              
ipdb> logging.disable(sys.maxsize)                                                                                                                                                           
ipdb> logging.info("silent mode")                                                                                                                                                    
ipdb> logging.disable(logging.NOTSET)                                                                                                                                                        
ipdb> logging.info("Enable logging again")                                                                                                                                                    
08:35:32 - MainThread - root - INFO - Enable logging again